### DISCUSSION TITLE
Steam Deck Trackpad Haptics not working

### DISCUSSION THREAD

**Discussion by hectormaximillian** [Valkirie/HandheldCompanion#589] [Q&A]
Hello, is theres a way to enable steam deck trackpad haptics? the left and right one doesnt vibrate when i swipe around, browsing files and stuff. Is theres an option to enable it?

**Comment by CasperH2O** [comment]
There is an open ticket for this item: https://github.com/Valkirie/HandheldCompanion/issues/590

Closing this as duplicate.
